index.html

```diff
@@ -87,17 +87,33 @@ <h1>Areanna Whittingotn</h1>
             <h2>Quotes</h2>
             <p>Just a girl trying to be successful</p>
             <p>To be beautiful means to be yourself. You don't need to be accepted by others. You need to accept yourself</p>
-            <ol>
-                <li>
-                        <a href="https://www.facebook.com/areyscreations">Facebook</a>
-                </li>
-                <li>
-                    <a href="https://www.instagram.com/areyscreations/">Instagram</a>
-                </li>
-                <li>
-                    <a href="http://www.areys.online">Website</a>
-                </li>
-            </ol>
+                 <h2>Contact</h2>
+                <ol>
+                    Email: areys.online@gmail.com
+                    <li>
+                         <a href="http://www.facebook.com/areyscreations"> 
+                            <i class="fab fa-facebook"></i>
+                             Facebook
+                         </a>
+                    </li>
+                     <li>
+                         <a href="http://www.instagram.com/areyscreations"> 
+                             <i class="fab fa-instagram"></i>
+                             Instagram
+                        </a>
+                     </li>
+                     <li> 
+                         <a href="http://www.areys.online">
+                            <i class="fas fa-home"></i>
+                             Website
+                         </a> 
+                     </li>
+                </ol>
+           
+<<<<<<< HEAD
```

Comment:
Here be a merge conflict! Make sure that you remove lines 113, 115, 116, and 141